Formula/swi-prolog.rb

```diff
@@ -69,7 +69,7 @@ def install
     # './prepare' prompts the user to build documentation
     # (which requires other modules). '3' is the option
     # to ignore documentation.
-    system "echo '3' | ./prepare" if build.head?
+    system "echo", "'3' | ./prepare" if build.head?
```

Comment:
Yes, please. Our audit check is unfortunately unable to fully understand this line, so it gives bad advice. Would be nice if we had an equivalent to `pipe_output` (often used in `test do`) that we could use in the `install` method for cases like these.